package.json

```diff
@@ -20,15 +20,15 @@
     "grunt-parallel": "^0.4.1",
     "grunt-subgrunt": "~1.1.0",
     "http-proxy": "^1.13.2",
-    "dce-paella-extensions": "1.6.16",
+    "dce-paella-extensions": "harvard-dce/dce-paella-extensions#t/MATT-2282-is-playing-live",
```

Comment:
Yes, as discussed we'll use a github version tag, e.g. `harvard-dce/dce-paella-extensions#v1.6.17`